src/ui/categories/common/CategoryFolderIcon.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
This is basically a copy of PageFolderIcon, why not moving PageFolderIcon to common directory, renaming it in "TreeNodeFolderIcon" and reuse it in both places?